testing/mocker/create_common.py

```diff
@@ -0,0 +1,150 @@
+import pprint
+from abc import abstractmethod
+
+import multiprocessing
+from typing import List
+from loguru import logger
+
+
+class CreateSubject(multiprocessing.Process):
+    """
+    This is s the abstract class that highlights the main way to interact with subject related creation process
+    """
+
+    def __init__(
+        self,
+        process_id,
+        index_subject,
+        path_to_mock_data,
+        list_sessions,
+        list_anatomy_modalities,
+    ):
+        """
+        Constructor that focus on create all sessions/modalities for a given subject.
+        :param process_id:
+        :param index_subject:
+        :param path_to_mock_data:
+        :param list_sessions:
+        :param list_anatomy_modalities:
+        """
+        super(CreateSubject, self).__init__()
+        self.id = process_id
+        self.index_subject = index_subject
+        self.path_to_mock_data = path_to_mock_data
+        self.list_sessions = list_sessions
+        self.list_bids_details = list_anatomy_modalities
+        self.list_bids_keywords = ["acq", "desc", "run"]
```

Comment:
I think that this variable is unused?
If not, what is it doing? why were those specific keywords hardcoded and not other?